pkg/sql/colexec/insert/insert.go

```diff
@@ -96,6 +97,10 @@ func Call(_ int, proc *process.Process, arg any) (bool, error) {
 
 		}
 	}
+	// XXX The original logic was buggy and I had to temporarily circumvent it
```

Comment:
Well ...   Why was buggy?   What is Zs?   Why does this one help?

Hack it fine, but please explain.

Reply:
ok，the upper-level code may not be populated with zs. As for zs, I will follow up to figure out how to deal with it.